lib/docker_service.rb

```diff
@@ -14,6 +14,34 @@ def self.wait_for_healthy_services(services: [], except: [])
     raise "Container(s) #{unhealthy_container_names.join(',')} were unhealthy after 60 seconds" if unhealthy_containers.any?
   end
 
+  def self.remove_built_app_images
+    built_images = built_app_images
+
+    built_images.each(&:remove)
+
+    built_images.map do |image|
+      image.info["RepoTags"].first
+    end
```

Comment:
You might want to just run the remove in the map and save iterating through the collection another time.